packages/block-library/src/block/block.json

```diff
@@ -8,6 +8,9 @@
 	"keywords": [ "reusable" ],
 	"textdomain": "default",
 	"attributes": {
+		"dynamicContent": {
```

Comment:
If we plan to keep it as an experiment, then it should get injected from outside, similar to `uses_context` for block types that have block bindings defined.